lib/text/vtt_text_parser.js

```diff
@@ -439,6 +440,77 @@ shaka.text.VttTextParser = class {
     }
   }
 
+  /**
+   * Converts voice style tag to be valid for xml parsing
+   * For example,
+   * input: <v Shaka>Test
+   * output: <v.voice-Shaka>Test</v.voice-Shaka>
+   *
+   * @param {string} payload
+   * @return {string} processed payload
+   * @private
+   */
+  static replaceVoiceStylePayload_(payload) {
```

Comment:
I don't know if this method is correct?
I tried running it on this simple content:
```
'WEBVTT\n\n' +
'00:00:20.000 --> 00:00:40.000\n' +
'<v A>Test\n\n' +
'00:00:40.000 --> 00:00:50.000\n' +
'<v B>Test'
```
And I got this output (broken into multiple lines for readability):
```
'WEBVTT\n\n' +
'00:00:20.000 --> 00:00:40.000\n' +
'<v.voice-A>Test\n\n' +
'00:00:40.000 --> 00:00:50.000\n' +
'<v.voice-B>Test\n' +
'</v.voice-A></v.voice-B>'
```
Note that both of the end tags are at the end of the file. Also, the end tags are in the wrong order; this ends up being like `<A><B></A></B>`, which is incorrectly nested.

Incidentally, if one of the tags already has an end tag, the other isn't given one:
```
'WEBVTT\n\n' +
'00:00:20.000 --> 00:00:40.000\n' +
'<v A>Test</v>\n\n' +
'00:00:40.000 --> 00:00:50.000\n' +
'<v B>Test'
```
Results in:
```
'WEBVTT\n\n' +
'00:00:20.000 --> 00:00:40.000\n' +
'<v.voice-A>Test</v.voice-A>\n\n' +
'00:00:40.000 --> 00:00:50.000\n' +
'<v.voice-B>Test\n'
```

Reply:
You are wrong! The payload parameter is `<v A>Test</v>` or `<v B>Test`. I retested with this input and it works well!

Reply:
Oh, I see, I had misread the method. I was mistakenly thinking it was `parseCueStyles` as in, parse the styles of all cues, not parse all of the styles of one cue.
I guess if the expectation is that there will be at most one voice tag per payload, this should be fine.